web/themes/custom/subtheme_videogame/src/components/field/_mediaquery.scss

```diff
@@ -0,0 +1,46 @@
+@media screen and (max-width: 576px) {
+  .user {
```

Comment:
Please place these styles in the relevant component's file instead of creating a separate file for media queries.